apps/portals/ampals/src/pages/HomePage.tsx

```diff
@@ -64,6 +68,17 @@ export default function HomePage() {
       </SectionLayout>
       {/* <AMPALSExploreTheData sql={upsetPlotSql} /> */}
       <HowToAccessData />
+      <SectionLayout
+        ContainerProps={{
+          className: 'home-spacer',
+          maxWidth: false,
+          sx: {
+            pb: '50px',
+          },
+        }}
+      >
+        <PortalFeaturedPartners sql={partnersSql} backgroundColor={'white'} />
```

Comment:
IMO move the background color to the `SectionLayout` sx and remove the prop. Would also have to update the EL portal